cmake/CallbackSetupExtraOptions.cmake

```diff
@@ -6,7 +6,7 @@ SET(CMAKE_MODULE_PATH  ${CMAKE_MODULE_PATH} "${Trilinos_SOURCE_DIR}/cmake")
 # Install TriBITS so that other projects can use it.
 INSTALL(
   DIRECTORY "${Trilinos_SOURCE_DIR}/cmake/tribits"
-  DESTINATION "${${PROJECT_NAME}_INSTALL_LIB_DIR}/cmake"
+  DESTINATION "${${CMAKE_PROJECT_NAME}_INSTALL_LIB_DIR}/cmake"
```

Comment:
Does this really fix this?  I have a fix about to push (that was on the release branch already).  Not sure how that happened.